### PR TITLE
ML 3.2 k-means Clustering (D)

### LINKED ISSUE #4103: ML 3.2 k-means Clustering (D)
### 📌  Issues for Week 3
Welcome to 'ML' Team, good to see you here

:arrow_forward: This issue will helps readers in acquiring all the knowledge that one needs to know about ----
### **_k-means Clustering_**.

:red_circle: To get assigned to this issue, add your **Batch Numbers** mentioned in the spreadsheet of "Machine Learning", the approach one would follow and choice you prefer (Documentation, Audio, Video). **Please note this issue is for DOCUMENTATION contribution only**

✔️ Domain : **Machine Learning**

:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.

- "Issue Title" and "PR Title should be the same. Include issue number along with it.

- Changes should be made inside the `Machine Learning/Machine_Learning/Unsupervised_Machine_Learning` Branch.

- Follow Contributing Guidelines & Code of Conduct before start Contributing.

- This issue is only for 'GWOC' contributors of 'Machine Learning' domain.

************************************************************

:white_check_mark: **To be Mentioned while taking the issue :**
- Full name
- Batch Number
- GitHub Profile Link
- Which type of Contribution you want to make : 
    - [ ] Documentation

******************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

### Domain

Machine Learning

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by deepthi1107**
<hr>

## Description 📜

documentation explaining k-means Clustering.
Fixes #4103

<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] Machine Learning

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by deepthi1107** [comment]
Hello @abhisheks008 sir, please do evaluate my PR.
Name: Deepthi M
Batch no. : 02
Serial Number: 172

File: Machine_Learning/Unsupervised_Machine_Learning/k-means_Clustering_(D).md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+## k-means Clustering
+
+Clustering is used to group or cluster the homogeneous groups. Homogeneous groups are formed by using distance measures.
+
+**K-means Clustering:**
+
+K-means clustering is an unsupervised machine learning model. It is used to group similar data values. The similarity is measured by distance.
+It transforms te unlabeled data to labeled data.
```
Comment (Isha307) [inline Machine_Learning/Unsupervised_Machine_Learning/k-means_Clustering_(D).md:8]: te ->the

**Review by Isha307 — COMMENTED** [review]
Everything else is fine.

**Comment by deepthi1107** [comment]
I have made changes mam @Isha307

**Review by Isha307 — APPROVED** [review]

**Pull request merged**